README.md

```diff
@@ -143,7 +143,7 @@ For adding support for other cloud providers, open an issue or propose a PR.
                 zone: eu-central-2a
               count: 2
               serverType: t3.medium
-              image: ami-0965bd5ba4d59211c
+              image: ami-0618becf24e8d2b5f
```

Comment:
```suggestion
              image: ami-0e4d1886bf4bb88d5
```

According to https://cloud-images.ubuntu.com/locator/ec2/, this should be the correct AMI for Ubuntu 22.04 AMD in eu-central-2

Reply:
@Despire please check